Rock.Rest/Controllers/FinancialTransactionsController.Partial.cs

```diff
@@ -317,7 +317,33 @@ public DataSet GetContributionTransactions( int groupId, int? personId, [FromBod
         }
 
         /// <summary>
-        /// 
+        /// Gets the contribution transactions.
+        /// </summary>
+        /// <param name="groupId">The group unique identifier.</param>
+        /// <returns></returns>
+        /// <exception cref="System.Web.Http.HttpResponseException"></exception>
+        [Authenticate, Secured]
+        [HttpGet]
+        [EnableQuery]
+        [System.Web.Http.Route( "api/FinancialTransactions/GetTransactionsForGivingGroup/{personId}" )]
+        public IQueryable<FinancialTransaction> GetTransactionsForGivingGroup( int personId )
+        {
```

Comment:
@azturner Yes our app depends on this to show giving history.  The idea was that it would return all transactions for the giving group that the supplied person is a part of.  I thought the GivingId was the same concept as the GivingGroupId?  I just checked with @jbaxleyiii and the GivingId is not currently mapped in the client app.  

I'll be glad to call it whatever you want so long as we are all clear on what it does (it sounds like I may be confused about the GivingGroupId vs GivingId)